test/features/bootstrap_options.feature

```diff
@@ -0,0 +1,44 @@
+@bootstrap
+Feature: HA bootstrap process - options
```

Comment:
I still miss documentation here. What's the meaning of options? A short description after the name would be helpful.
We can add docstring always to improve the test purpose.

Reply:
changed